app/external/rawls.py

```diff
@@ -17,12 +17,13 @@ def get_workspace_uuid_and_project(workspace_namespace: str, workspace_name: str
         f"{os.environ.get('RAWLS_URL')}/api/workspaces/{workspace_namespace}/{workspace_name}?fields=workspace.workspaceId,workspace.googleProject",
         headers={"Authorization": bearer_token})
 
+    jso = resp.json()
     if resp.ok:
-        jso = resp.json()
         return RawlsWorkspaceResponse(workspace_id=jso["workspace"]["workspaceId"], google_project=jso["workspace"]["googleProject"])
     else:
         # just pass the error upwards
-        logging.info(f"Got {resp.status_code} from Rawls for {workspace_namespace}/{workspace_name}: {resp.text}")
+        logging.info(f"Got {resp.status_code} from Rawls for {workspace_namespace}/{workspace_name}: {jso['message']}",
+                     extra={"json_fields": jso})
```

Comment:
since this is just a proof of structured logging support, how about something that doesn't risk exceptions, like:
```suggestion
        workspace_dict = { 'workspace': { 'namespace': workspace_namespace, 'name': workspace_name} }
        logging.info(f"Got {resp.status_code} from Rawls for {workspace_namespace}/{workspace_name}: {jso['message']}",
                     extra={"json_fields": workspace_dict})
```

Reply:
Done.  And the swatomation run worked this time too (see below).